lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = ActiveModel::Serializer::IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
Does this need to be in the initializer?  Does it need to be eager evaluated? Also, you could

```ruby
def initialize(*)
  super
  # use whatever the super method did
```